front-end/src/shared/SideBarNav.js

```diff
@@ -18,7 +23,7 @@ class SideBarNav extends React.Component {
             {/* Commenting this out for now so that the customer doesn't get confused seeing this when logged in */}
             {/* <li><Link to="/signup">Signup</Link></li> */}
             {/* TODO: Implement both back end and front end */}
-            <li><Link to="/">Sign Out</Link></li>
+            <li onClick={this.handleSignOut}><Link to ="#">Sign Out</Link></li>
```

Comment:
Could we just navigate them back to login on signout?

Reply:
We could... but if they navigated to any other page again, they would still be logged in. We need to actually drop the JWT to force the application to treat the user as if he/she is not authenticated.